### PR TITLE
cathy/sponsors2

### LINKED ISSUE #26: our sponsors pt.2
image

this is on the home page

absolute positioning is needed

### PR DESCRIPTION
**Pull request by catheiwu**
absolute positioning for "our sponsors" and "blueprint test prep" on home page
figma
sponsors

### CODE REVIEW

**Review by stanleylew5 — CHANGES_REQUESTED** [review]
image

hm i merged dev into your branch cos it was outdated and something broke feel free to look for it and fix it

File: src/components/home/Header.jsx [hunk 1]
```diff
@@ -26,6 +26,16 @@ const Header = () => {
           <p className="text-[13rem] w-[30%] leading-none mt-[-1.5%]">LAW</p>
         </div>
       </div>
+
+      <div className="relative bg-witl-purple-300 mt-[50%] w-[100%]">
+        <div className="text-witl-purple-200 pl-[12%] flex flex-col items-start leading-tight">
+          <div className="text-[8rem] mt-[5%]">OUR</div>
+          <div className="text-[14rem]">&nbsp;&nbsp;SPONSORS</div>
```
Comment (stanleylew5) [inline src/components/home/Header.jsx]: use text-10xl instead

File: src/components/home/Header.jsx [hunk 2]
```diff
@@ -26,6 +26,16 @@ const Header = () => {
           <p className="text-[13rem] w-[30%] leading-none mt-[-1.5%]">LAW</p>
         </div>
       </div>
+
+      <div className="relative bg-witl-purple-300 mt-[50%] w-[100%]">
+        <div className="text-witl-purple-200 pl-[12%] flex flex-col items-start leading-tight">
```
Comment (stanleylew5) [inline src/components/home/Header.jsx]: just make it a tad smaller pl[10%]

**Review by stanleylew5 — CHANGES_REQUESTED** [review]

**Comment by catheiwu** [comment]
Screen Shot 2024-07-09 at 11 12 55 AM
Used text-10xl instead of text-[14rem], decreased padding

**Pull request merged**